packages/now-build-utils/src/fs/glob.ts

```diff
@@ -41,11 +45,11 @@ export default async function glob(pattern: string, opts: GlobOptions | string,
   const files = await vanillaGlob(pattern, options);
 
   for (const relativePath of files) {
-    const fsPath = path.join(options.cwd!, relativePath);
+    const fsPath = path.join(options.cwd!, relativePath).replace(/\\/g, '/');
```

Comment:
I'm worried about the implications of this change, is it really necessary?
```suggestion
    const fsPath = path.join(options.cwd!, relativePath);
```

Reply:
@Timer That's just only because of Windows, my tests didn't work so I asked @TooTallNate if he was cool with me adding it and he said yes

Reply:
I'm just worried because we just shipped `now dev` windows support and want to make sure we didn't break it!

Reply:
@lfades Does this hit the `statCache` correctly on Windows? I would assume this would break and cause `statCache does not contain value` to be hit.